knative-serving/config-applications/pvcs-for-serving.adoc

```diff
@@ -4,8 +4,14 @@ include::_attributes/common-attributes.adoc[]
 = Persistent Volume Claims for Serving
 :context: pvcs-for-serving
 
-Some serverless applications need permanent data storage. 
-To achieve this, you can configure persistent volume claims (PVCs) for your Knative services.
+Some serverless applications need permanent data storage. You can provide data storage for Knative services by configuring different volume types. Serving supports mouthing the volume types such as `secret`, `configMap`, `projected`, and `emptyDir`. 
+
+You can configure persistent volume claims (PVCs) for your Knative services. The PersistentVolume types are implemented as plugins. Persistent volumes are supported but require a feature flag that must be enabled. Serverless applications also support PersistentVolume types such as `awsElasticBlockStore`, `gcePersistentDisk`, and `cephfs`.
```

Comment:
I've put a comment on https://issues.redhat.com/browse/SRVKS-1077?focusedId=23518023&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-23518023 with some details around that. We should probably add more details and suggest using OpenShift Data Foundation.

Reply:
I've just noticed that we already mention `ocs-storagecluster-cephfs`.
It would be good explain that it's coming from OpenShift Data Foundation as a tested solution: https://docs.openshift.com/container-platform/4.14/storage/persistent_storage/persistent-storage-ocs.html

Reply:
Just to summarize the previous comments and discussion on slack:
* We should remove the sentence "Serverless applications also support PersistentVolume types such as `awsElasticBlockStore`, `gcePersistentDisk`, and `cephfs`." as these were not tested downstream.
* We should explaing that the `ocs-storagecluster-cephfs` storage class being used in the example later in this document is coming from OpenShift Data Foundation and provide a link to docs.